business_register/models/declaration_models.py

```diff
@@ -119,6 +119,12 @@ def destroy(self):
     def nacp_url(self):
         return f'https://public.nazk.gov.ua/documents/{self.nacp_declaration_id}'
 
+    @property
+    def declared_resignation_at(self):
+        if self.type == Declaration.BEFORE_RESIGNATION:
+            return self.submission_date
+        return
```

Comment:
Explicit is better than implicit
```suggestion
        return None
```